create_infotexts.py

```diff
@@ -100,19 +101,42 @@ def load_places_mapping():
 
     return places_dict
 
+
+def create_commons_filename(metadata, fotonr):
+    """Creates commons filename according to https://phabricator.wikimedia.org/T156612 and ouputs to new_dict
+    :param metadata: dictionary
+    :param fotonr: string representing the <Fotonummer> field in the metadata
+    """
+    if not metadata[fotonr]["Beskrivning"] == "":
+        cleaned_fname = helpers.format_filename(metadata[fotonr]["Beskrivning"], "SMVK-MM-Cypern",
+                                                metadata[fotonr]["Fotonummer"])
+        # print("Fname using BatchUploadTools: {}".format(cleaned_fname))
+    else:
+        # TODO: fix alternative description according to https://phabricator.wikimedia.org/T156612#3008806
```

Comment:
I think this is where you were supposed to link to the task for generating a better description.